src/sage/schemes/elliptic_curves/hom.py

```diff
@@ -452,6 +452,29 @@ def kernel_polynomial(self):
         """
         raise NotImplementedError('children must implement')
 
+    def kernel_points(self):
+        """
+        Return an iterator over the points in the kernel of this
+        elliptic-curve morphism.
+
+        TESTS::
```

Comment:
```suggestion
        EXAMPLES::
```
Since this is public, it should have at least one doctest in the online doc (`TESTS::` are (annoyingly IMO) hidden).